### ISSUE TITLE
Module not loading

### ISSUE THREAD

**Issue by aunrea** [tylerstambaugh/MMM-StravaWeekInBike#28]
I only see the title of the module, no content. Looking at the loaded code the div where the module content goes is empty.

In the console I have this error:
`NodeHelper log: "MMM-StravaWeekInBike - Node helper getStravaStats - Error fetching data from API: AxiosError: Request failed with status code 401`

**Comment by tylerstambaugh** [comment]
@aunrea A 401 indicates an issue with the token. Did you follow the steps Strava outlines for getting a bearer token? Are you using any other MM modules that use your Strava bearer token? Are you able to use Postman or similar tool to verify the token is valid and has the required scopes?

**Comment by aunrea** [comment]
Yes, I followed the instructions. I have no other module using the token. Postman says the token is valid and has the read-all scope. Am I supposed to use the authorization code with the module? It's not listed in the example config.

**Comment by aunrea** [comment]
How am I supposed to do step 11? It's not clear with your module.

**Comment by tylerstambaugh** [comment]
Are you referencing step 11 on [Strava's instructions](https://developers.strava.com/docs/getting-started/#account)?

**Comment by aunrea** [comment]
Yes. I am not sure how I apply it to your modules.
